internal/test/smoke/system_endpoints_test.go

```diff
@@ -87,20 +95,60 @@ func (s *SuiteSystemEndpoints) TestHostConfExecute() {
 	t := s.T()
 	s.As(RBACSuperAdmin)
 	s.prepareDomainIpa(t)
-
-	t.Log("Calling SystemHostConfWithResponse")
 	domainType := public.RhelIdm
 	s.As(XRHIDSystem, RBACNoPermis)
-	res, err := s.HostConfWithResponse(
-		s.domain.RhelIdm.Servers[0].SubscriptionManagerId.String(),
-		"client."+s.domain.DomainName,
-		builder_api.NewHostConf().
-			WithDomainName(pointy.String(s.domain.DomainName)).
-			WithDomainType(&domainType).
-			Build())
-	require.NoError(t, err)
-	require.NotNil(t, res)
-	require.Equal(t, http.StatusOK, res.StatusCode)
+
+	t.Run("HostConf success", func(t *testing.T) {
+		// Expect success HostConf event to be sent to Pendo
+		mockPendo, ok := s.PendoClient.(*mock_pendo.Pendo)
+		require.True(t, ok)
+		mockPendo.On("SendTrackEvent", mock.Anything, mock.MatchedBy(func(r *pendo.TrackRequest) bool {
+			return (r.Event == "HostConf" &&
+				r.AccountID == s.systemXRHID.Identity.OrgID &&
+				r.VisitorID == s.systemXRHID.Identity.System.CommonName)
+		})).Return(nil)
+
+		res, err := s.HostConfWithResponse(
+			s.domain.RhelIdm.Servers[0].SubscriptionManagerId.String(),
+			"client."+s.domain.DomainName,
+			builder_api.NewHostConf().
+				WithDomainName(pointy.String(s.domain.DomainName)).
+				WithDomainType(&domainType).
+				Build())
+		require.NoError(t, err)
+		require.NotNil(t, res)
+		err = res.Body.Close()
+		require.NoError(t, err)
+		require.Equal(t, http.StatusOK, res.StatusCode)
+
+		mockPendo.AssertExpectations(t)
+	})
+
+	t.Run("HostConf failure", func(t *testing.T) {
+		// Expect failure HostConf event to be sent to Pendo
+		mockPendo, ok := s.PendoClient.(*mock_pendo.Pendo)
+		require.True(t, ok)
+		mockPendo.On("SendTrackEvent", mock.Anything, mock.MatchedBy(func(r *pendo.TrackRequest) bool {
+			return (r.Event == "HostConfFailed" &&
```

Comment:
Similar to the above; by moving the constants to a common place, we would use here too, and any change to the value would not impact the test.